cdc/log.hh

```diff
@@ -123,7 +123,7 @@ struct db_context final {
 enum class operation : int8_t {
     // note: these values will eventually be read by a third party, probably not privvy to this
     // enum decl, so don't change the constant values (or the datatype).
-    pre_image = 0, update = 1, row_delete = 2, range_delete_start = 3, range_delete_end = 4, partition_delete = 5
+    pre_image = 0, insert = 1, update = 2, row_delete = 3, range_delete_start = 4, range_delete_end = 5, partition_delete = 6
```

Comment:
Won't this cause mis-interpretation of existing data? Maybe append to the end instead?

Reply:
We're doing backward incompatible changes anyway. Note we're still `experimental`.
Even if I appended it at the end, it would still be a backward-incompatible change (because previously `insert`s were treated as `update`s)

Reply:
We will still need to be able to update from a previous experimental version. Although maybe it is acceptable to ask the user to truncate.

Reply:
Even without this PR we won't be able to update 3.4 from 3.3 without disabling CDC on all tables first. (even truncate is not enough)